var/spack/repos/builtin/packages/emacs/package.py

```diff
@@ -59,8 +60,6 @@ class Emacs(AutotoolsPackage):
     depends_on('libxaw', when='+X toolkit=athena')
     depends_on('gtkplus', when='+X toolkit=gtk')
     depends_on('gnutls', when='+tls')
-    depends_on('libxpm ^gettext+libunistring', when='+tls')
-    depends_on('ncurses+termlib', when='+tls')
```

Comment:
What's up with this?

Reply:
patience, I'm in the middle of typing up the explanation.....